README.md

```diff
@@ -1,24 +1,106 @@
- <img src="https://raw.githubusercontent.com/noisetorch/NoiseTorch/master/assets/icon/noisetorch.png" width="100" height="100">
+<h1 align="center"> NoiseTorch</h1>
+<h3 align="center"> Noise Supression Application for PulseAudio or Pipewire</h3>
+<p align="center"><img src="https://raw.githubusercontent.com/noisetorch/NoiseTorch/master/assets/icon/noisetorch.png" width="100" height="100"></p> 
 
-# NoiseTorch
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0) [![Last Release](https://img.shields.io/github/v/release/noisetorch/NoiseTorch?label=latest&style=flat-square)](https://github.com/noisetorch/NoiseTorch/releases) 
```

Comment:
Can you center the badges? I think it would look better.

Reply:
while I share that sentiment, that centered badges may look good, it is not a necessity to get this PR merged.
Thanks for the changes, @RiemaruKarurosu